### ISSUE TITLE
Support AMQP cluster for events endpoints

### ISSUE THREAD

**Issue by okamototk** [ManageIQ/manageiq-providers-openstack#295]
When use AMQP for event integration, we can specify just one host.
Basically, AMQP  is clustered for high availability.  So multiple AMQP host shoud be specified at provider configuration.

**Comment by aufi** [comment]
Hi, thanks for your message. We do recommend using Ceilometer/Panko for Events, which should fit to HA OpenStack deployments.

Adding multiple AMQP hosts should be possible, but I'm not sure about its development priority. If you're interested in this feature I'd encourage you try work on it, https://github.com/ManageIQ/guides/blob/master/developer_setup.md

**Comment by okamototk** [comment]
Thinks,

We don't use Ceilometer and don't want to install Ceilometer for ManageIQ purpose only.
Currently we have just PoC and this priority is not so high. 
But this feature is one of the important point to judge we use ManageIQ or not for production.

If we prefer other ManageIQ features perfectly, I'll look to fix problem. 
However currently, I hope ManageIQ guy will implement this feature.

**Comment by alexander-demichev** [comment]
https://github.com/ManageIQ/manageiq-providers-openstack/pull/394

This PR adds this feature on backend side

**Comment by alexander-demichev** [comment]
@aufi Can we close this issue?

**Comment by JPrause** [comment]
Removing the stale label as this is still active.
@miq-bot remove_label stale

**Comment by aufi** [comment]
This feature was completed. Add provider form now includes two AMQP fallback hostname fields. See https://github.com/ManageIQ/manageiq-ui-classic/pull/5270 for details.

**Comment by maly-glod** [comment]
Hi @aufi, 

While browsing this thread, I came across information that two AMQP fallback hostname fields were added. I currently have the Petrosian version of ManageIQ, and I don't see these additional fields. From which official (stable) version of ManageIQ did these changes come into effect?

![Screenshot 2024-07-10 at 17 32 03](https://github.com/ManageIQ/manageiq-providers-openstack/assets/111138186/3825ea4e-148b-4c05-bc5d-495e37f9a332)
